packages/components/src/Tooltip/tests/chromatic/Tooltip.stories.tsx

```diff
@@ -0,0 +1,242 @@
+                <Tooltip {...args} />
+            </TooltipTrigger>
+        </Stack>
+    ),
+    decorators: [
+        Story => (
+            <Flex justifyContent="left">
+                <Story />
+            </Flex>
+        )
+    ],
+    parameters: {
+        skipGlobalDecorator: true
+    }
+} satisfies Story;
+
+export const LinkTrigger = {
+    render: args => (
+        <TooltipTrigger defaultOpen>
+            <Link>{buttonText}</Link>
+            <Tooltip {...args} />
+        </TooltipTrigger>
+    )
+} satisfies Story;
+
+export const AvatarTrigger = {
+    render: function Render(args) {
+        return (
+            <TooltipTrigger defaultOpen>
+                <PassiveTrigger>
+                    <Avatar name="Fred Allen" />
+                </PassiveTrigger>
+                <Tooltip {...args} />
+            </TooltipTrigger>
+        );
+    }
+} satisfies Story;
+
+export const LongContent = {
+    render: args => (
+        <TooltipTrigger isOpen>
+            <Button>{buttonText}</Button>
+            <Tooltip {...args} />
+        </TooltipTrigger>
+    ), 
+    args: {
+        children: `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam in turpis
+                        ac libero tincidunt hendrerit. Ut vitae nisl nec orci laoreet tristique.
+                        Nulla facilisi. Nulla facilisi. Nulla facilisi. Nulla facilisi. Nulla
+                        facilisi. Nulla facilisi. Nulla facilisi. Nulla facilisi. Nulla facilisi.`
+    },
+    decorators: [
+        Story => (
+            <Flex UNSAFE_marginTop="8rem" justifyContent="center">
+                <Story />
+            </Flex>
+        )
+    ],
+    parameters: {
+        skipGlobalDecorator: true
+    }
+} satisfies Story;
+
+export const DisabledTrigger = {
+    render: args => (
+        <TooltipTrigger>
+            <PassiveTrigger data-testid="passive-trigger">
+                <Button isDisabled>{buttonText}</Button>
+            </PassiveTrigger>
+            <Tooltip {...args} />
+        </TooltipTrigger>
+    ),
+    play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
+        const trigger = canvas.getAllByTestId("passive-trigger")[0];
+        const trigger2 = canvas.getAllByTestId("passive-trigger")[1];
+        // For some reason, we need to hover over the second trigger first
+        await userEvent.hover(trigger2);
+        await userEvent.hover(trigger);
+        await waitFor(async () => {
+            await expect(screen.getByText(childrenText)).toBeVisible();
+        });
+    }
+} satisfies Story;
+
+export const DisabledListItem = {
+    render: args => (
+        <TooltipTrigger>
+            <ListBox selectionMode="multiple">
+                <ListBoxItem id="1" isDisabled position="relative">
+                    <PassiveTrigger data-testid="passive-trigger" width="auto" position="absolute" top="0" bottom="0" right="0" left="0" />
```

Comment:
Oh no this synthax is really not appropriate for our DS. 
i was expecting something a lot more similar to Orbiter

```
<Select placeholder="Planets">
    <TooltipTrigger>
        <ListBoxItem key="earth">Earth</ListBoxItem>
        <Tooltip>Home sweet home!</Tooltip>
    </TooltipTrigger>
    <TooltipTrigger>
        <ListBoxItem key="jupiter">Jupiter</ListBoxItem >
        <Tooltip>The fourth-brightest object in the sky</Tooltip>
    </TooltipTrigger>
    <TooltipTrigger>
        <ListBoxItem key="mars">Mars</ListBoxItem >
        <Tooltip>See you in 2026</Tooltip>
    </TooltipTrigger>
</Select>
```

Reply:
you can't have anything between a list item and ListBox. 
Also, when I put TooltipTrigger inside the ListBoxitem, there was also an error.
I can try again to see if there's anything to be done, but I'm not sure.

Reply:
There is lots of possible solution for this. 

We can put the tooltip trigger inside the content ourselve inside the ListboxItem component
We can create a ListBoxItemTooltipTrigger component that is an item with a tooltip
We can drop to the hook level and create our own ListboxItem if necessary. 

It's a matter of making a plan and investigating the different avenues.